website-html-to-markdown/codemods/transforms/refactor-properties.js

```diff
@@ -119,7 +124,7 @@ module.exports = function ({ source, path }, { parse, visit }) {
           let output = [];
           output.push(build.text(`<api>`));
           properties.forEach((p) => {
-            output.push(build.text(`<output data-name="${p['name']}" data-type="${p['type']?p['type']:'–'}" data-value="${p['value']?p['value']:'–'}"${p['default']?' data-default="'+p['default']+'"':''}${p['required']?' data-required="true"':''}>`));
+            output.push(build.text(`<output data-name="${p['name']}" data-type="${p['type']?p['type']:''}" data-value="${p['value']?p['value']:''}"${p['default']?' data-default="'+p['default']+'"':''}${p['required']?' data-required="true"':''}>`));
             output.push(build.text(`${p['notes']?p['notes']:'–'}`));
```

Comment:
@didoo I still had to keep the `-` here otherwise, if the tag has no content, it gets stripped out by turndown entirely (including the attributes), but I remove it afterward when outputting the Ember code, so all good in the end.